configs/pie-components-config/vite.config.js

```diff
@@ -22,7 +22,7 @@ const sharedConfig = ({ build = {}, plugins = [], ...rest }) => defineConfig({
         },
         rollupOptions: {
             external: (id) => {
-                if (id === 'react' || /^lit/.test(id)) {
+                if (id === 'react' || /^lit/.test(id) || id === '@lit/react') {
```

Comment:
this is the main change, the rest are just readme updates to use the new package name :D

Reply:
Not tested but i think you can shorten this a bit to make it a bit more readable:

`['react', '@lit/react'].includes(id) || /^lit/.test(id)`

Reply:
I've thought about that but `includes` isn't strict and one of the id entries is the `react.ts` file which shouldn't be marked as external.

Reply:
ahh yea makes sense